From 13f5535ab20756548e0528908dfd9052b202a03e Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 6 May 2019 17:02:26 +0800
Subject: [PATCH] arm64: dts: rockchip: move the common things to the
 rk3399-linux.dtsi

In order to reduce the duplication of code, let's move the common things
to the rk3399-linux.dtsi.

Change-Id: I066d99642190cfcbe1c8d16d05849297176f9e6b
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
index 129e5a58aae3..38dccdddfaaa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Fuzhou Rockchip Electronics Co., Ltd
+ * Copyright (c) 2019 Fuzhou Rockchip Electronics Co., Ltd
  *
  * This file is dual-licensed: you can use it either under the terms
  * of the GPL or the X11 license, at your option. Note that this dual
@@ -75,6 +75,17 @@
 		memory-region = <&ramoops_mem>;
 	};
 
+	fiq_debugger: fiq-debugger {
+		compatible = "rockchip,fiq-debugger";
+		rockchip,serial-id = <2>;
+		rockchip,signal-irq = <182>;
+		rockchip,wake-irq = <0>;
+		rockchip,irq-mode-enable = <1>;  /* If enable uart uses irq instead of fiq */
+		rockchip,baudrate = <1500000>;  /* Only 115200 and 1500000 */
+		pinctrl-names = "default";
+		pinctrl-0 = <&uart2c_xfer>;
+	};
+
 	cif_isp0: cif_isp@ff910000 {
 		compatible = "rockchip,rk3399-cif-isp";
 		rockchip,grf = <&grf>;
@@ -206,6 +217,10 @@
 	status = "okay";
 };
 
+&uart2 {
+	status = "disabled";
+};
+
 &pinctrl {
 	isp {
 		cif_clkout: cif-clkout {
-- 
2.35.3

